models/bob.js

```diff
@@ -22,18 +23,43 @@ const BobModel = mongoose.model('Bob', bobSchema);
   @param {Object[]} bobData
 */
 function saveBob(bobData) {
+  let mediaStatus = "";
+  if(bobData.data.Link) {
+    mediaStatus = false;
+  } else {
+    console.log("Warning: Saving bob without Link");
```

Comment:
This block assumes all bobs should have links, is this true? Do we still care about text and whatnot?